<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="thanksDisplay">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
      <fontweight>Medium</fontweight>
     </font>
    </property>
    <property name="text">
     <string>Thankyou for your Business!</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>491</width>
      <height>462</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_9">
     <item>
      <widget class="QLabel" name="label_13">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>INVOICE GENERATOR</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Issued To : </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="issuedToLineEdit">
               <property name="minimumSize">
                <size>
                 <width>138</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>138</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="minimumSize">
                  <size>
                   <width>90</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Pay To : </string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="payToLineEdit">
                 <property name="minimumSize">
                  <size>
                   <width>138</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>138</width>
                   <height>16777215</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="label_6">
                 <property name="minimumSize">
                  <size>
                   <width>90</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Account Name : </string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="accNameLineEdit">
                 <property name="minimumSize">
                  <size>
                   <width>138</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>138</width>
                   <height>16777215</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="label_7">
                 <property name="minimumSize">
                  <size>
                   <width>90</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Account No. : </string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="accNoLineEdit">
                 <property name="minimumSize">
                  <size>
                   <width>138</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>138</width>
                   <height>16777215</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>INVOICE NO .:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="invoiceNumberLineEdit">
               <property name="maximumSize">
                <size>
                 <width>138</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>DATE :</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="dateLineEdit">
               <property name="maximumSize">
                <size>
                 <width>138</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_12">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>DUE DATE :</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="dueDateLineEdit">
               <property name="maximumSize">
                <size>
                 <width>138</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Orientation::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QTableWidget" name="tableWidget">
           <column>
            <property name="text">
             <string>DESCRIPTION</string>
            </property>
            <property name="font">
             <font>
              <bold>true</bold>
             </font>
            </property>
           </column>
           <column>
            <property name="text">
             <string>QTY</string>
            </property>
            <property name="font">
             <font>
              <bold>true</bold>
             </font>
            </property>
           </column>
           <column>
            <property name="text">
             <string>UNIT PRICE</string>
            </property>
            <property name="font">
             <font>
              <bold>true</bold>
             </font>
            </property>
           </column>
           <column>
            <property name="text">
             <string>TOTAL</string>
            </property>
            <property name="font">
             <font>
              <bold>true</bold>
             </font>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QPushButton" name="pushButtonAdd">
               <property name="text">
                <string>Add Item</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonDel">
               <property name="text">
                <string>Delete Item</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Orientation::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>Generate PDF</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="leftMargin">
            <number>6</number>
           </property>
           <property name="topMargin">
            <number>6</number>
           </property>
           <property name="rightMargin">
            <number>6</number>
           </property>
           <property name="bottomMargin">
            <number>6</number>
           </property>
           <item>
            <widget class="QLabel" name="labelSubTotal">
             <property name="minimumSize">
              <size>
               <width>75</width>
               <height>0</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>  SUBTOTAL :  </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="subTotalDisplay">
             <property name="minimumSize">
              <size>
               <width>70</width>
               <height>0</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="leftMargin">
            <number>6</number>
           </property>
           <property name="topMargin">
            <number>24</number>
           </property>
           <property name="rightMargin">
            <number>6</number>
           </property>
           <property name="bottomMargin">
            <number>6</number>
           </property>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>35</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <property name="rightMargin">
          <number>12</number>
         </property>
         <item row="0" column="1">
          <widget class="QLabel" name="displayTax">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>10%</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelTotal">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string> Total : </string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="labelTax">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>    Tax : </string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="displayTotal">
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>45</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>515</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
